libs/designsystem/header/src/public_api.ts

```diff
@@ -0,0 +1,2 @@
+export * from './header.component';
+export * from './header.module';
```

Comment:
This should be named exports.
See comment on `libs/designsystem/header/index.ts`